### ISSUE TITLE
Only first post alternate skin

### ISSUE THREAD

**Issue by SieBer15** [dudaster/ele-custom-skin#844]
How can I accomplish that only the first post is an alternate template using the function Alternate templates in Ele Custom Skin Pro?

**Comment by gabrielbarbera** [comment]
Did you found a solution for this? I need to do the same

**Comment by SieBer15** [comment]
Yes I used a workaround by using two postwidgets. The first widget you use loop 1 and only show one post, the second widget you use loop 2 and set an offset of one.

**Comment by gabrielbarbera** [comment]
Yes, I have done the same thing on other projects, but it will be ideal for
making this work with alternate templates. Thanks anyways for your answer,
will help others for sure.
